### PR TITLE
AC-1010 show body on error

### PR DESCRIPTION
**Pull request by dibericky**
Show responseBody when statusCode is 401.

### CODE REVIEW

File: packages/api-explorer/src/components/Response/ResponseBody.jsx [hunk 1]
```diff
@@ -100,14 +107,14 @@ Unauthorized.defaultProps = {
 function ResponseBody({ result, isOauth, oauth }) {
   return (
     <div style={{ display: 'block'}}>
-      {result.status !== 401 && <Authorized result={result} />}
-      {result.status === 401 && <Unauthorized isOauth={isOauth} oauth={oauth} />}
+      {result.status !== 401 && <Result result={result} />}
```
Comment (fredmaggiowski) [inline packages/api-explorer/src/components/Response/ResponseBody.jsx]: Simplify the whole component by checking for `401` and print the message. Always print the `div` for Result.

This way there's no need for an `Unauthorized` and `Result` component.

File: packages/api-explorer/src/components/Response/ResponseBody.jsx [hunk 2]
```diff
@@ -81,15 +82,19 @@ function hasOauth(oauth) {
...
 function Unauthorized({ isOauth, oauth }) {
   return (
-    <div className="text-center hub-expired-token">
-      {isOauth ? hasOauth(oauth) : <p><FormattedMessage id="api.auth.failed" defaultMessage="You couldn't be authenticated" /></p>}
-    </div>
+    <Fragment>
+      <div style={notJsonStyle}>
+        {isOauth ? hasOauth(oauth) : <p><FormattedMessage id="api.auth.failed" defaultMessage="You couldn't be authenticated" /></p>}
+      </div>
+
```
Comment (fredmaggiowski) [inline packages/api-explorer/src/components/Response/ResponseBody.jsx]: Remove empty line

File: packages/api-explorer/src/components/Response/ResponseBody.jsx [hunk 3]
```diff
@@ -100,14 +105,20 @@ Unauthorized.defaultProps = {
...
-      {result.status !== 401 && <Authorized result={result} />}
-      {result.status === 401 && <Unauthorized isOauth={isOauth} oauth={oauth} />}
+      {
+        result.status === 401 && <Unauthorized isOauth={isOauth} oauth={oauth} />
+      }
+      {
+        result.responseBody ? <Result result={result} /> 
+        : <FormattedMessage id={'response.noBody'} default={'The response has no body'} />
+      }
+
```
Comment (fredmaggiowski) [inline packages/api-explorer/src/components/Response/ResponseBody.jsx]: Remove empty line

File: packages/api-explorer/src/components/Response/ResponseBody.jsx (hunk 2 above)
Comment (dibericky) [reply packages/api-explorer/src/components/Response/ResponseBody.jsx]:
```suggestion
```

**Comment by fredmaggiowski** [comment]
LGTM

File: packages/api-explorer/src/components/Response/ResponseBody.jsx [hunk 4]
```diff
@@ -81,15 +82,18 @@ function hasOauth(oauth) {
 
 function Unauthorized({ isOauth, oauth }) {
   return (
-    <div className="text-center hub-expired-token">
-      {isOauth ? hasOauth(oauth) : <p><FormattedMessage id="api.auth.failed" defaultMessage="You couldn't be authenticated" /></p>}
-    </div>
+    <Fragment>
```
Comment (davidebianchi) [inline packages/api-explorer/src/components/Response/ResponseBody.jsx]: This is not necessary

File: packages/api-explorer/src/components/Response/ResponseBody.jsx [hunk 5]
```diff
@@ -81,15 +82,18 @@ function hasOauth(oauth) {
...
+        {isOauth ? hasOauth(oauth) : <p><FormattedMessage id="api.auth.failed" defaultMessage="You couldn't be authenticated" /></p>}
+      </div>
+    </Fragment>
   );
 }
 
 Unauthorized.propTypes = {
   isOauth: PropTypes.bool,
   oauth: PropTypes.bool.isRequired,
+  result: PropTypes.shape({}).isRequired
```
Comment (davidebianchi) [inline packages/api-explorer/src/components/Response/ResponseBody.jsx]:
```suggestion
  result: PropTypes.object.isRequired
```

**Pull request merged**